main.js

```diff
@@ -0,0 +1,252 @@
+// debugger;
+
+class Game {
+  constructor() {
+    this.board = [
+      [1, 2, 3],
+      [4, 5, 6],
+      [7, 8, 9]
+    ];
+  }
+  reset() {
+    this.board = [
+      [1, 2, 3],
+      [4, 5, 6],
+      [7, 8, 9]
+    ];
+    counter = 0;
+  }
+
+
+  checkWinner(gameBoard) {
+    var position1 = game.board[0][0];
+    var position2 = game.board[0][1];
+    var position3 = game.board[0][2];
+    var position4 = game.board[1][0];
+    var position5 = game.board[1][1];
+    var position6 = game.board[1][2];
+    var position7 = game.board[2][0];
+    var position8 = game.board[2][1];
+    var position9 = game.board[2][2];
```

Comment:
At the risk of being harsh here, this is bad practice to hard-code all of this. You have to keep track of sooo many variables which is incredibly tedious and very prone to all kinds of bugs.